ui/src/types/document.ts

```diff
@@ -17,6 +17,13 @@ export type ProjectDocument = {
   updates: number[];
 };
 
+export type PreviewSnapshot = {
```

Comment:
Kinda related to my other question, we now have ProjectSnapshot, ProjectDocument, and PreviewSnapshot. All basically the same, just ProjectSnapshot doesn't care about the id apparently. These should be reduced to a single type. If necessary, please discuss with @kasugamirai 🙏🏻